src/utils/queries.test.js

```diff
@@ -32,3 +32,97 @@ describe('getResource', () => {
     ).rejects.toThrow(errorMessage);
   });
 });
+
+describe('getResources', () => {
+  it('returns `data` object when successfully fetched', async () => {
+    const data = {
+      count: 2,
+      next: 'http://localhost:8000/api/v1/resources/?page=2',
+      previous: null,
+      results: [
+        {
+          guid: '4e85dbb6-d519-11ea-a622-0242ac130002',
+          author: '',
+          title: 'gitduck',
+          description: '',
+          url: 'https://gitduck.com',
+          referring_url: '',
+          other_referring_source: '',
+          user: {
+            id: 229,
+            username: 'aug2nd',
+            first_name: 'Linda',
+            last_name: 'Peng',
+            is_superuser: false,
+          },
+          date_published: '2020-08-02T16:38:45.651676-07:00',
+          created: '2020-08-02T16:38:45.652112-07:00',
+          modified: '2020-08-02T16:38:45.651710-07:00',
+          media_type: 'Podcast',
+          paid: true,
+          tags: [],
+        },
+        {
+          guid: '12f02bb2-e2aa-11ea-a58a-0242ac130006',
+          author: '',
+          title: 'google',
+          description: '',
+          url: 'https://google.com',
+          referring_url: '',
+          other_referring_source: '',
+          user: {
+            id: 233,
+            username: 'Austyn99',
+            first_name: 'Jarret',
+            last_name: 'Mitchell',
+            is_superuser: false,
+          },
+          date_published: '2020-08-19T22:57:47.828418-07:00',
+          created: '2020-08-19T22:57:47.831689-07:00',
+          modified: '2020-08-19T22:57:47.828464-07:00',
+          media_type: 'Video',
+          paid: true,
+          tags: [],
+        },
+      ],
+    };
+
+    axios.get.mockImplementationOnce(() => Promise.resolve({ data }));
+
+    await expect(getResources('')).resolves.toEqual(data);
+  });
+});
+it('returns the appropriate result when gitduck is searched', async () => {
+  const data = {
+    count: 1,
+    next: 'http://localhost:8000/api/v1/resources/?page=2',
+    previous: null,
+    results: [
+      {
+        guid: '4e85dbb6-d519-11ea-a622-0242ac130002',
+        author: '',
+        title: 'gitduck',
+        description: '',
+        url: 'https://gitduck.com',
+        referring_url: '',
+        other_referring_source: '',
+        user: {
+          id: 229,
+          username: 'aug2nd',
+          first_name: 'Linda',
+          last_name: 'Peng',
+          is_superuser: false,
+        },
+        date_published: '2020-08-02T16:38:45.651676-07:00',
+        created: '2020-08-02T16:38:45.652112-07:00',
+        modified: '2020-08-02T16:38:45.651710-07:00',
+        media_type: 'Podcast',
+        paid: true,
+        tags: [],
+      },
+    ],
+  };
+
+  axios.get.mockImplementationOnce(() => Promise.resolve({ data }));
```

Comment:
I find this to be a little weird but unsure how to really fix it. Just leaving this as a comment and perhaps something that we'll need to figure out how to do better. 

We're asserting on the response object that we implemented. If the API breaks and that the response object is different, we'll never know in this case.